data-gathering/bot.py

```diff
@@ -522,152 +522,152 @@ def analisi():
 
         try:
 
-          url = "https://"+lingua+".wikipedia.org/w/api.php?action=parse&page=" + discussioneURL + voce2 + "&prop=wikitext&formatversion=2&format=json"
+          url = "https://"+language+".wikipedia.org/w/api.php?action=parse&page=" + discussionURL + voice2 + "&prop=wikitext&formatversion=2&format=json"
           json_url = urlopen(url)
 
           data = json.loads(json_url.read())
 
-          wikitext_discussione = data["parse"]["wikitext"]
+          wikitext_discussion = data["parse"]["wikitext"]
 
         except:
 
-          wikitext_discussione = ""
+          wikitext_discussion = ""
 
 
 
-        ris = ris + voce + "\t"
+        res = res + voice + "\t"
 
-        ris = ris + wikidataid + "\t"
+        res = res + wikidataid + "\t"
 
       except:
 
-        ris = ris + voce +"\t" +"Voce inesistente"
+        res = res + voice +"\t" +"Voice inexistent"
 
        
 
       else:
 
-        if primoEdit:
+        if firstEdit:
 
-           ris = ris + primoEdit(voce2) + "\t"
+           res = res + firstEdit(voice2) + "\t"
 
 
 
-        if dimensione:
+        if dimension:
 
-           ris = ris + dimensione(wikitext) + "\t"
+           res = res + dimension(wikitext) + "\t"
 
 
 
-        if immagini:
+        if images:
 
-           ris = ris + immagini(wikitext) + "\t"
+           res = res + images(wikitext) + "\t"
 
 
 
         if note:
 
-           ris = ris + note(wikitext) + "\t"
+           res = res + note(wikitext) + "\t"
 
            
 
-        if configAvvisi:
+        if configAlerts:
```

Comment:
and here you say configAlerts

Reply:
hi @EneHacheCe @wikicurricula-uy this has now been fixed